src/components/PopUps/SelectContact.js

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+import XButton from "../Buttons/XButton";
+import Contact from "../../assets/Contact.png";
+import ButtonPurple from "../Buttons/ButtonPurple";
+
+const SelectContact = ({ isVisible, onClose }) => {
+  if (!isVisible) return null;
+  
+
+  return (
+    <PopUpContainer>
+      <CloseButton onClick={onClose}>
+        <XButton />
+      </CloseButton>
+      <ContactIcon src={Contact} alt="Contact Icon" />
+      <ContactName>Mom</ContactName>
+      <ButtonPurple title="Chat" />
+      <AboutText>About</AboutText>
+      <DescriptionText>Guardian*</DescriptionText>
+      <DescriptionText>MY MOMMMM xD</DescriptionText>
+      <Spacer />
+      <EmailText>Email</EmailText>
+      <EmailValue>mom@gmail.com</EmailValue>
+    </PopUpContainer>
+  );
+};
+
+export default SelectContact;
+
+const PopUpContainer = styled.div`
+  position: fixed;
+  top: 20%; /* Adjust this value to move it vertically */
+  right: 150px; /* Set right distance from the right edge */
+  width: 500px;
+  height: 500px;
+  background-color: white;
+  color: black;
+  border-radius: 8px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start; /* Align items to the start vertically */
+  z-index: 1000; /* Ensures the popup is on top */
```

Comment:
ChatGPT doesn't know you haven't used z-index elsewhere. I believe 2 would do, like above ;)